docs/github/contribute_to_repos.md

```diff
@@ -0,0 +1,57 @@
+       > Add a new section to reports called ‘Best Practices Violations’. It should be inserted immediately after and appear just like the existing critical validation errors table and include validation notices W1, W2, and W3.
+
+* When closing an issue without a merged PR, please indicate one of the following (they provide a paper trail for reporting and indicate lessons learned):
+   * What non-PR actions were taken to resolve the issue
+   * Why no action is necessary
+   * Why it is no longer an issue at this time
+
+
+## Working on Branches & Creating Pull Requests:
+
+
+* Organize your work into [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+* Commit and push early and often for team visibility.
+* Split up a large PR if it’s getting too big (code reviews should ideally take no more than 10 minutes).
+* PR description is required and should document the purpose of the PR in a way that someone who was not involved at the time can understand later.
+   * For example, the following description is not sufficient as persistent PR documentation (though it may be sufficient very temporarily while a critical fix is in flight with multiple people collaborating closely and reviewing each others' work).
+      > Hotfix - type2
+   * The persistent description should generally be at least as detailed as:
+      > Hotfix for issue where merge_updates.py throws a MadeUpError exception on files containing the number 7 (issue was identified on ITP ID 1 URL 1, feed_info file extracted on 2022-03-22).
+      Fix was to add a try/except block to catch the number 7 and cast it to string before running the merge SQL. More investigation information is documented in [link to Slack thread].
+
+
+* PRs should be linked to any issues that they close. [Keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) are one good way to do this.
+
+
+## Code Reviews - PR Author:
+
+
+* All PRs should get one approving review before being merged.
+* Move your PR out of draft status when it’s ready for review and merging.
+* Maintain responsibility for all PRs you open until they are merged, keeping an eye out for comments and merging promptly after approval is received.
+* PR authors should merge PRs once approval is received.
+
+
+## Code Reviews - Reviewer:
+* Reviewers should not submit comments on PRs that are still in draft status unless specifically requested by the PR author.
+* Practice [conventional comments](https://conventionalcomments.org/) to make it clear to recipients of review what notes require what action, and what’s just feedback.
+* If you are approving a PR and leave a comment with your approval, the PR author should understand that that comment is **optional** to address.
+   * If you have a change that you think should be **required** before the change is merged, please request changes on the PR rather than approving.
```

Comment:
google has some decent advice on how to leave review comments https://google.github.io/eng-practices/review/reviewer/

Reply:
**thought (belated):** I like Google's `eng-practices` doc, particulary the section on 'What to look for in a code review'. James and Mjumbe have been following https://conventionalcomments.org/ when leaving PR comments. I mistakenly thought this was linked in the README under `Contributing`, but only [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) is listed. 

**suggestion (non-blocking):** Could we direct folks to Google's [How to do a code review](https://google.github.io/eng-practices/review/reviewer/) recommendations for their reference, and link https://conventionalcomments.org/ for leaving actionable comments in PRs? (_I am also pretty new to this format -- I hope I'm doing this right!)_